src/fluree/db/json_ld/transact.cljc

```diff
@@ -382,7 +384,7 @@
 
 (defn insert
   "Performs insert transaction"
-  [{:keys [schema] :as db} json-ld opts]
+  [{:keys [schema issuer] :as db} json-ld opts]
```

Comment:
Is this issuer binding used in this fn?

Reply:
No, but it is used by consumers if there is one.

Reply:
This is minor, but if it isn't used by the current function then I think the binding should either not be there or be prefixed with an '_' if you want to include it in the binding list for documentation purposes.